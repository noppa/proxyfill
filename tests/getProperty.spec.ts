import traverse from './traverse'

describe('proxyfill Babel plugin', () => {
	it('should transform getters', () => {
		expect(traverse('foo.bar')).toMatchSnapshot()
	})
})
