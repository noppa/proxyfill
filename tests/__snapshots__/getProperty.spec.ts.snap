// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proxyfill Babel plugin should transform getters 1`] = `
"import {
    get as _proxyfillRuntime$get,
    set as _proxyfillRuntime$set,
    invoke as _proxyfillRuntime$invoke,
} from 'proxyfill/runtime'

_proxyfillRuntime$get(foo, 'bar')
"
`;
